raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py

```diff
@@ -103,102 +121,114 @@ def make_route_from_channelstate(channel_state):
 
 def test_is_lock_valid():
     """ A hash time lock is valid up to the expiration block. """
-    expiration = 10
-    assert mediator.is_lock_valid(expiration, 5) is True
-    assert mediator.is_lock_valid(expiration, 10) is True, "lock is expired at the next block"
-    assert mediator.is_lock_valid(expiration, 11) is False
+    expiration = BlockExpiration(10)
+    assert mediator.is_lock_valid(expiration, BlockNumber(5)) is True
+    assert (
+        mediator.is_lock_valid(expiration, BlockNumber(10)) is True
+    ), "lock is expired at the next block"
+    assert mediator.is_lock_valid(expiration, BlockNumber(11)) is False
 
 
 def test_is_safe_to_wait():
     """ It's safe to wait for a secret while there are more than reveal timeout
     blocks until the lock expiration.
     """
-    expiration = 40
+    expiration = BlockExpiration(40)
 
     # expiration is in 30 blocks, 19 blocks safe for waiting
-    block_number = 10
-    reveal_timeout = 10
+    block_number = BlockNumber(10)
+    reveal_timeout = BlockTimeout(10)
     assert mediator.is_safe_to_wait(expiration, reveal_timeout, block_number).ok
 
     # expiration is in 20 blocks, 10 blocks safe for waiting
-    block_number = 20
-    reveal_timeout = 10
+    block_number = BlockNumber(20)
+    reveal_timeout = BlockTimeout(10)
     assert mediator.is_safe_to_wait(expiration, reveal_timeout, block_number).ok
 
     # expiration is in 11 blocks, 1 block safe for waiting
-    block_number = 29
-    reveal_timeout = 10
+    block_number = BlockNumber(29)
+    reveal_timeout = BlockTimeout(10)
     assert mediator.is_safe_to_wait(expiration, reveal_timeout, block_number).ok
 
     # at the block 30 it's not safe to wait anymore
-    block_number = 30
-    reveal_timeout = 10
+    block_number = BlockNumber(30)
+    reveal_timeout = BlockTimeout(10)
     assert mediator.is_safe_to_wait(
         expiration, reveal_timeout, block_number
     ).fail, "this is expiration must not be safe"
 
-    block_number = 40
-    reveal_timeout = 10
+    block_number = BlockNumber(40)
+    reveal_timeout = BlockTimeout(10)
     assert mediator.is_safe_to_wait(
         expiration, reveal_timeout, block_number
     ).fail, "this is expiration must not be safe"
 
-    block_number = 50
-    reveal_timeout = 10
+    block_number = BlockNumber(50)
+    reveal_timeout = BlockTimeout(10)
     assert mediator.is_safe_to_wait(
         expiration, reveal_timeout, block_number
     ).fail, "this is expiration must not be safe"
 
 
 def test_is_channel_usable_for_mediation():
     """ Check rules that determine if a channel can be used for transfers """
-    reveal_timeout = 30
-    timeout_blocks = reveal_timeout + 10
-    amount = UNIT_TRANSFER_AMOUNT
+    reveal_timeout = BlockTimeout(30)
+    timeout_blocks = BlockTimeout(reveal_timeout + 10)
+    amount = TokenAmount(UNIT_TRANSFER_AMOUNT)
```

Comment:
This shouldn't be necessary

Reply:
Yes, I got confused by a test that used this as an expiration value.